src/esphomelib/time/ds1307.cpp

```diff
@@ -52,8 +52,7 @@ bool DS1307Time::read_data() {
   t.tm_min  = ds1307_.reg.minute + 10 * ds1307_.reg.minute_10;
   t.tm_sec  = ds1307_.reg.second + 10 * ds1307_.reg.second_10;
   struct timeval tv {::mktime(&t), 0};
-  timezone tz = {0, 0};
-  ::settimeofday(&tv, &tz);
+  ::settimeofday(&tv, 0);
```

Comment:
Revert this please. If you do not specify the second argument for `settimeofday`, the timezone will be +9h china time. (The SDK is not fully POSIX compliant, took me a while to figure that out for home assistant time.) - timezones are handled by the layer above settimeofday in the TZ env-variable.